### PR TITLE
Implement paper wallet on registration - Closes #1594

### LINKED ISSUE #1594: Implement paper wallet in registration
Subtask of https://github.com/liskhq/lisk-hub/issues/1574

- The second step of registration should contain a button to generate and download pdf paper wallet.

- The pdf generation can be solved like here: https://codepen.io/fasterv_410/pen/OXwNLd

### PR DESCRIPTION
**Pull request by renatomassao**
### What issue have I solved?

#1594 

### How have I implemented/fixed it?

Used jsPDF lib to generate the PDF, had to render the QRCode on the page, so it was able to pass the base64 data to the lib to put it into the PDF.

### How has this been tested?

Go to `/register` and select an avatar, and on the next step click on the `download PDF` and check that the date, address and passphrase are correct, also the value of the QR code should be the address

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: src/components/registerV2/backupPassphrase.js [hunk 1]
```diff
@@ -90,33 +108,30 @@ class BackupPassphrase extends React.Component {
...
+                {t('Paper version')}
+                <Tooltip
+                  title={'Paperwallet'}>
+                  <p>
+                    {t('You can print your passphrase to be stored in a safe place. ')}
+                    {t('It is highly recommended to delete the PDF and remove it from your trash after printing.')}
+                  </p>
+                </Tooltip>
+              </h2>
+              <div style={{ display: 'none' }} ref={this.setCanvasRef}>
```
Comment (michaeltomasik) [inline src/components/registerV2/backupPassphrase.js]: Is that div necessary? It is not displayed anyway in the app

File: src/utils/paperwallet.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    return this;
+  }
+
+  save() {
+    this.renderHeader()
+      .renderInstructions()
+      .renderAccount()
+      .renderFooter()
+      .doc.output('dataurlnewwindow');
+    // .doc.save('Lisk.pdf');
```
Comment (michaeltomasik) [inline src/utils/paperwallet.js]: please remove unused code

**Comment by renatomassao** [comment]
@Efefefef Done

**Comment by Efefefef** [comment]
👍

**Pull request merged**